website/static/js/nodesPrivacy.js

```diff
@@ -139,9 +143,10 @@ var NodesPrivacyViewModel = function(node, onSetPrivacy) {
             return MESSAGES.makeEmbargoPublicTitle;
         }
 
-
         return {
-            warning: 'Warning',
+            warning: self.parentIsPublic ?
+                self.isComponent ? 'Make component private' : 'Make project private' :
```

Comment:
On L111 you are casting the word 'project' and 'component into a boolean, and here you are checking the boolean to decide whether to use the word 'project' or 'component. You could probably simplify this significantly.